### DISCUSSION TITLE
Welcome to Nominatim Discussions!

### DISCUSSION THREAD

**Discussion by lonvia** [osm-search/Nominatim#2094] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. This is the right place to:
  * Ask questions you’re wondering about.
  * Ask for help with your own installation of Nominatim.
  * Share ideas.
  * Engage with other community members.

We hope that you welcome others and are open-minded. Remember that this is a community we build together.

**Comment by otbutz** [comment]
Similar to graphhoppers new issue [templates](https://github.com/graphhopper/graphhopper/issues/new/choose): 

https://github.com/osm-search/Nominatim/pull/2096
